R/brreg-utils.R

```diff
@@ -67,7 +67,7 @@ parse_brreg_entity_single <- function(p) {
       p$forretningsadresse$postnummer, p$forretningsadresse$poststed)),
     kommune = null2na(p$forretningsadresse$kommune),
     postadresse = null2na(sprintf(
-      '%s, %s %s', unlist(p$postadresse$adresse),
+      '%s, %s %s', unlist(paste0(p$postadresse$adresse, collapse = '; ')),
```

Comment:
Documentation of the function should be expanded with the following under details: "`get_entity()` will return a single row for each entity. If one or more columns return multiple values, the values will be separated with a semicolon `;`."